### ISSUE TITLE
Dropping text (or element) throws TypeError: Cannot read property 'type' of undefined

### ISSUE THREAD

**Issue by Tomino2112** [danialfarid/ng-file-upload#1280]
Hi,

If I just select some text on the page or take some element and drag it into the drop area, instead of some graceful error or validation I get error thrown:

```
TypeError: Cannot read property 'type' of undefined
    at ng-file-upload-all.min.js:4
    at angular.js:14792
    at r.$eval (angular.js:16052)
    at r.$digest (angular.js:15870)
    at r.$apply (angular.js:16160)
    at angular.js:17927
    at e (angular.js:5552)
    at angular.js:5829
```

**Comment by mmrvka** [comment]
I'm getting this error on Ubuntu Chrome Version 47.0.2526.111 (64-bit) also on the demo page http://angular-file-upload.appspot.com/ (which is actually ng-file-upload 11.1.1 at time of this writing) when dropping a file into the "Select File or Drop File" box.

```
TypeError: Cannot read property 'type' of undefined
    at ng-file-upload.js:1949
    at wrappedCallback (angular.js:11572)
    at angular.js:11658
    at Scope.$eval (angular.js:12701)
    at Scope.$digest (angular.js:12513)
    at Scope.$apply (angular.js:12805)
    at angular.js:14313
    at completeOutstandingRequest (angular.js:4397)
    at angular.js:4705
```

The reason seems to be that dataTransfer.files and dataTransfer.items are empty.

On Ubuntu Firefox 43.0.4, it works as expected.

**Comment by mmrvka** [comment]
Seems to be related to this: https://code.google.com/p/chromium/issues/detail?id=543940 (at least my issue).

**Comment by danialfarid** [comment]
Fixed at 11.2.0
